.github/workflows/release.yml

```
@@ -0,0 +1,92 @@
+name: release
+on:
+  push:
+    tags:
+    # match semver versions
+    - "v[0-9]+.[0-9]+.[0-9]+"
+    # match semver pre-releases
+    - "v[0-9]+.[0-9]+.[0-9]+-*"
+jobs:
+  release:
+    env:
+      NETLIFY_BASE: 'videojs-preview.netlify.app/sandbox/'
```

I think we actually want to link to the index with all the other demos here?
```suggestion
      NETLIFY_BASE: 'videojs-preview.netlify.app'
```

---

_I guess so_